connectors/v2/anghami.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+/* global Connector */
+
+
+Connector.artistSelector = '.track-artist';
+
+Connector.trackSelector = '.track-title';
+
+Connector.currentTimeSelector = '.position';
+
+Connector.durationSelector = '.duration';
+
+Connector.playButtonSelector = '.play-pause-button paused';
```

Comment:
My bad, what I meant to do was `'.play-pause-button.paused'` instead ([selector chaining](http://stackoverflow.com/a/3772305/1499431)). I'm not exactly sure why it was working for me before, at least it didn't start scrobbling till I hit the play button even if the pause button was visible.

Anyway, I'm not sure if overriding `isPlaying` is better or going with chained classes selector is. I will go with the selector now, but let me know what you think.

**Paused**
![anghami-paused](https://cloud.githubusercontent.com/assets/5252681/21591783/d72836ce-d10f-11e6-817c-4eb713b0c1a3.png)

**Playing**
![anghami-playing](https://cloud.githubusercontent.com/assets/5252681/21591784/d7285528-d10f-11e6-88f0-a5bcab65e3ec.png)